react/index.d.ts

```diff
@@ -2140,60 +2140,237 @@ declare namespace React {
         unselectable?: boolean;
     }
 
+    // this list is "complete" in that it contains every SVG attribute 
+    // that React supports, but the types can be improved.
+    // Full list here: https://facebook.github.io/react/docs/dom-elements.html
```

Comment:
This link is useful, but I'm not sure what the intent of the comment above is. Do you mean it as a `TODO`? If so, how?